django/utils/deprecation.py

```diff
@@ -2,9 +2,12 @@
 import warnings
 
 
-class RemovedInDjango20Warning(PendingDeprecationWarning):
+class RemovedInDjango110Warning(PendingDeprecationWarning):
     pass
 
+# for backwards compatibility in Django 1.8.3
+RemovedInDjango20Warning = RemovedInDjango110Warning
```

Comment:
I think I would do `RemovedInDjango20Warning = PendingDeprecationWarning` so we don't give it a meaning it doesn't really have.